contentcuration/contentcuration/views/files.py

```diff
@@ -223,3 +223,11 @@ def debug_serve_file(request, path):
     with default_storage.open(filepath, 'rb') as fobj:
         response = HttpResponse(FileWrapper(fobj))
         return response
+
+
+def debug_serve_content_database_file(request, path):
+    filename = os.path.basename(path)
+    path = "/".join([settings.DB_ROOT, filename])
+    with default_storage.open(path, "rb") as f:
+        response = HttpResponse(FileWrapper(f))
```

Comment:
`response = HttpResponse(FileWrapper(f), content_type='application/octet-stream')` should be used or contet-type will be ` text/html;` what is wrong when downloding a sqlite file.

Otoh, we need to add 
`'django.middleware.http.ConditionalGetMiddleware',` to the list of MIDDLEWARE_CLASSES in contencuration/settings.py file to get the content-length information of the file. Without that information Kolibri task to download the database can't work.

Reply:
Good catch!